engine/src/main/java/io/camunda/zeebe/engine/processing/streamprocessor/writers/EventApplyingStateWriter.java

```diff
@@ -15,7 +15,7 @@
  * A state writer that uses the event applier, to alter the state for each written event.
  *
  * <p>Note that it does not write events to the stream itself, but it delegates this to the {@link
- * TypedStreamWriter}.
+ * LegacyTypedStreamWriter}.
```

Comment:
🔧 I can't put this comment at a different line as that code is unchanged, so I'll put it here instead. I believe this class can be removed entirely. It's only been used in tests currently.

Reply:
Good catch!